frontend/src/app/components/NavigationMenu/NavigationMenu.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import MenuItem from './MenuItem/MenuItem';
+import NavigationLogo from './Logo/NavigationLogo';
+
+import { signOut } from '../../../../auth';
+
+import './NavigationMenu.scss';
+
+interface IMenuItems {
+    href: string;
+    textToRender: string;
+}
+
+const menuItemsArr: IMenuItems[] = [
+    { href: '/dashboard', textToRender: 'Přehled' },
+    { href: '/dashboard/mytips', textToRender: 'Moje tipy' },
+    { href: '/dashboard/currentresults', textToRender: 'Průběžné výsledky' },
+    { href: '/dashboard/overallresults', textToRender: 'Celkové výsledky' },
+    { href: '/dashboard/profile', textToRender: 'Profil' },
+];
+
+const NavigationMenu: React.FC = () => {
+    return (
+        <main className='mainContainer'>
+            <div className='logoContainer'>
+                <NavigationLogo />
+            </div>
+            <div className='menuContainer'>
+                {menuItemsArr.map((item, index) => {
+                    return (
+                        <MenuItem key={index} href={item.href} textToRender={item.textToRender} />
+                    );
+                })}
+                <form
+                    // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
Also forgotten?

Reply:
Well, not exactly. I prefer to keep this one here for the moment. TS doesn't like this way of handle of promise, but ima gonna fix that in future, I hope

Reply:
Cool, then you can ignore my poor knowledge